datasets/migrations/0094_auto_20210916_2301.py

```diff
@@ -0,0 +1,68 @@
+# Generated by Django 2.2.4 on 2021-09-17 03:01
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('datasets', '0093_auto_20210916_0007'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='ocahousingcourt',
+            name='classification',
+            field=models.TextField(blank=True, null=True),
+        ),
+        migrations.AddField(
+            model_name='ocahousingcourt',
+            name='court',
+            field=models.TextField(blank=True, null=True),
+        ),
+        migrations.AddField(
+            model_name='ocahousingcourt',
+            name='dateofjurydemand',
+            field=models.TextField(blank=True, null=True),
```

Comment:
Should this be a datefield?

Reply:
The data we have is either empty or inconsistent for the datefield to be used.